app/javascript/admin/controllers/article_controller.js

```diff
@@ -85,6 +85,8 @@ export default class ArticleController extends Controller {
     const [, , xhr] = event.detail;
     newArticle.innerHTML = xhr.responseText;
 
-    this.element.innerHTML = newArticle.querySelector('.card').innerHTML;
+    this.element.innerHTML = newArticle.querySelector(
+      '.js-individual-article',
```

Comment:
I added a -js class here as we were depending on the classname `.card` which changed when we changed the markup. We haven't yet established a best practice for js related classnames but we have had conversations about it. It's on my to do list to discuss with team members again, but for now this isn't adding a new pattern.